spec/semantics.html

```diff
@@ -763,7 +772,7 @@ <h2></h2>
       </section>
       <section>
         <h2>Basic entailment</h2>
-The previous definition allows us to introduce the notion of basic entailment:
+The previous definition allows us to introduce the notion of basic entailment:~
```

Comment:
If this tilde was intentional, please explain it, at least in comment.
```suggestion
The previous definition allows us to introduce the notion of basic entailment:
```